src/app/components/referral/referral-dashboard.tsx

```diff
@@ -77,23 +81,23 @@ export const ReferralDashboard: React.FC<Props> = (props) => {
   }, [props.tree]);
 
   const info = React.useMemo(() => {
-    if (!props.tree || !team || !props.address || !props.claimed) return;
+    if (!logs || !props.tree || !team || !props.address || !props.claimed || !referralTree) return;
 
     const stats = calculateStats(logs);
+    const myRefCodeStats = stats[referralTree?.id];
 
-    const subs = team.map((x) => stats[x.id]);
-    const subStats = subs.filter((x) => !!x);
```

Comment:
There is a chance, that the sublead has no purchases, so without filtering it will hit undefined